src/modules/dailyCreditsSummary/i18n/fr_FR.ts

```diff
@@ -0,0 +1,102 @@
+        },
+        cancelledMissions: {
+            regex: '^%missions%( \\(Fire Alarm System\\))? - Annulée$',
+            title: 'Missions annulées',
+            backgroundColor: '#7C7978',
+            textColor: 'white',
+        },
+        buyVehicle: {
+            regex: /Véhicule acheté/,
+            backgroundColor: '#007fff',
+            textColor: 'white',
+        },
+        dailyLogin: {
+            regex: /Récompense de connexion quotidienne/,
+            backgroundColor: '#8db600',
+            textColor: 'black',
+        },
+        buildings: {
+            regex: /(Poste construit|Refund Building$|Building demolished without refund)/,
+            title: 'Poste construit/demolished',
+            backgroundColor: '#ed872d',
+            textColor: 'black',
+        },
+        upgradeBuilding: {
+            regex: /(Station (upgraded( \(from small .*? station\))??|constructed)|Extended guard|Cancel:|Refund Building upgrade)/,
+            title: 'Station constructed/extended',
+            backgroundColor: '#000080',
+            textColor: 'white',
+        },
+        task: {
+            regex: /Tâche accomplie : («.*?»|".*?")/,
+            title: 'Tâche accomplie',
+            backgroundColor: '#bb3385',
+            textColor: 'white',
+        },
+        patients: {
+            regex: /Traitement de patient/,
+            title: 'Patients',
+            backgroundColor: '#fff600',
+            textColor: 'black',
+        },
+        alliancePatients: {
+            regex: /Hopital - Alliance/,
+            backgroundColor: '#ffff99',
+            textColor: 'black',
+        },
+        alliancePrisoners: {
+            regex: /Police - Alliance/,
+            backgroundColor: '#ffff99',
+            textColor: 'black',
+        },
+        prisoners: {
+            regex: /Prisonnier transporté/,
+            title: 'Prisoner',
+            backgroundColor: '#0bda51',
+            textColor: 'black',
+        },
+        schoolings: {
+            regex: /(education|Alliance Training Applicant)/,
+            title: 'Education',
+            backgroundColor: '#ff00ff',
+            textColor: 'white',
+        },
+        shitComplexes: {
+            regex: /(Attached building to Building Complex Base|Upgraded to Building Complex Base|Upgraded to Building Complex|Building Complex Base upgrade finished)/,
+            title: 'Building Complex',
+            backgroundColor: '#b5651d',
+            textColor: 'white',
+        },
+        fireAlarmSystemCancel: {
+            regex: /False Alarm/,
+            titel: 'False Alarms',
+            backgroundColor: '#c80815',
+            textColor: 'white',
+        },
+    },
+    category: 'Categorie',
+    title: 'Sommaire',
+    total: 'Total',
+    others: 'Others',
+    export: {
+        export: 'Export',
+        json: {
+            raw: 'JSON (raw)',
+            prettified: 'JSON (prettified)',
+        },
+    },
+};
```

Comment:
The regexes must match what's standing in https://www.operateur112.fr/credits/daily would be good if you could check that.
However: Any entry that does not match will be in category "others", so it is not the most important